components/ActionSheet/src/MDCActionSheetController.h

```diff
@@ -140,8 +140,29 @@ __attribute__((objc_subclassing_restricted))
  */
 @property(nonatomic, nullable, strong) UIFont *actionFont;
 
+/**
+ The color applied to the sheet view of the action sheet controller.
+ */
 @property(nonatomic, nonnull, strong) UIColor *backgroundColor;
 
+/**
+ The color applied that will be applied to message if there is a message. If there is no message
```

Comment:
This doesn't work with themers. If I add these properties as well as primary and secondary text color then they may become out of sync.

Reply:
Please remove `primaryTextColor` and `secondaryTextColor`

Reply:
I can remove `titleTextColor` and `messageTextColor` because those change based on each other but I cannot remove `primaryTextColor` and `secondaryTextColor`.